installing/installing_gcp/installing-gcp-shared-vpc.adoc

```diff
@@ -17,9 +17,9 @@ The installation program provisions the rest of the required infrastructure, whi
 * You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
 * You read the documentation on xref:../../installing/overview/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
 * If you use a firewall, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured it to allow the sites] that your cluster requires access to.
-* You have a GCP host project which contains a shared VPC network.
-* You xref:../../installing/installing_gcp/installing-gcp-account.adoc#installing-gcp-account[configured a GCP project] to host the cluster. This project, known as the service project, must be attached to the host project. For more information, see link:https://cloud.google.com/vpc/docs/provisioning-shared-vpc#create-shared[Attaching service projects in the GCP documentation].
-* You have a GCP service account that has the xref:../../installing/installing_gcp/installing-gcp-account.adoc#minimum-required-permissions-ipi-gcp-xpn_installing-gcp-account[required GCP permissions] in both the host and service projects.
+* You have a {gcp-short} host project that contains a shared VPC network and that has a configured Cloud Network Address Translation (NAT) resource.
```

Comment:
Suggest to replace "_and that has a configured Cloud Network Address Translation (NAT) resource_" with "_and that has a configured Cloud Router and Cloud NAT gateway to ensure Internet access from the VPC is available_".